### PR TITLE
WIP: Add role to install and configure icingaweb2

### PR DESCRIPTION
**Pull request by l00ptr**
Hi;

this is a first MR to add icingaweb2 role. This one allows to install icingaweb2, initialize the database and manage some config (ldap integration, command transport,...).

Let me know if it's something useful for you and how to fit your code / ansible habits ?!

Regards,
L.

### CODE REVIEW

**Comment by martialblog** [comment]
Hi, thanks for the PR! We'll review it as soon as possible. 

Relates to #31

File: roles/icingaweb2/tasks/config_commandtransports.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+- name: ICINGAWEB | Create monitoring module folder"
+  ansible.builtin.file:
+    path: "/etc/icingaweb2/modules/monitoring/"
+    state: directory
+    owner: "{{ icingaweb2_webserver_user }}" 
+    group: icingaweb2
```
Comment (martialblog) [inline roles/icingaweb2/tasks/config_commandtransports.yml:7]: The group should be a variable.

**Review by martialblog — CHANGES_REQUESTED** [review]
Hi, 

I left some comments. There are some things that need improvement.

Feel free to push changes to the branch. We will review them.

All the best,
Markus

File: roles/icingaweb2/tasks/config_commandtransports.yml (hunk 1 above)
Comment (l00ptr) [reply roles/icingaweb2/tasks/config_commandtransports.yml]: fixed! I can squash the commit if requested

File: roles/icingaweb2/tasks/config_db_backend.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    dest: /etc/icingaweb2/resources.ini
+    owner: "{{ icingaweb2_webserver_user }}" 
+    group: icingaweb2
+    mode: 0640
+    marker: ";{mark} ANSIBLE MANAGED BLOCK ICINGAWEB DB RESOURCES"
+    create: true
+    block: |
+      [icingaweb_db]
+      type = "db"
+      db = "pgsql"
```
[the remark this answers is not in the document]
Comment (l00ptr) [reply roles/icingaweb2/tasks/config_db_backend.yml:13]: can we make the db type a param first (on this PR) and then support mysql install / config within another PR ?

**Comment by mkayontour** [comment]
Hi, we decided to rewrite the role from scratch, there are many things to consider to make it usable for everybody. You're more than welcome to test the new role on main and give feedback how it is working.

**Pull request closed without merging**